Add release notes template (#731)
freedomofpress/dangerzone: docs/templates/release-notes-regular.md

> This release includes various new features, stability improvements, and security fixes. If you are on a Mac or PC you should also update Docker Desktop for security reasons.

Reviewer: While reading this sentence, I think that we can improve the wording a bit for subsequent releases:

```suggestion
This release includes various new features, stability improvements, and security fixes **(adjust accordingly)**. If you are on a Mac or PC please also update Docker Desktop to the latest version to get the latest security fixes
```

The highlights for this release are:
  - **<!--Highlight 1-->**
    <!-- highlight description (issue link) -->
  - **<!--Highlight N-->**

>     <!-- highlight description (issuegit link)-->

Reviewer: ```suggestion
    <!-- highlight description (issue link)-->
```

>   - **<!--Highlight 1-->**
>     <!-- highlight description (issue link) -->
>   - **<!--Highlight N-->**
>     <!-- highlight description (issuegit link)-->

Reviewer: I would propose converting all the above comments to actual text, so that people can look at the rendered Markdown from the GitHub UI, without having to check the raw Markdown file.

Also, I propose having examples in this list like:

```suggestion
  - **Important accomplishment**
    We used to do [this](https://github.com/freedomofpress/dangerzone/issues/1), but now we do [that](https://github.com/freedomofpress/dangerzone/issues/2).
  - **Support for a new platform**
    We added support for a new platform ([#3](https://github.com/freedomofpress/dangerzone/issues/3))
```

This way, it's easy to recall what tense do we use, how we link to issues, etc.
